qdisc.go

```diff
@@ -281,19 +281,22 @@ type Fq struct {
 	Buckets          uint32
 	FlowRefillDelay  uint32
 	LowRateThreshold uint32
+	Horizon          uint32
+	HorizonDrop      uint8
 }
 
 func (fq *Fq) String() string {
 	return fmt.Sprintf(
-		"{PacketLimit: %v, FlowPacketLimit: %v, Quantum: %v, InitialQuantum: %v, Pacing: %v, FlowDefaultRate: %v, FlowMaxRate: %v, Buckets: %v, FlowRefillDelay: %v,  LowRateThreshold: %v}",
-		fq.PacketLimit, fq.FlowPacketLimit, fq.Quantum, fq.InitialQuantum, fq.Pacing, fq.FlowDefaultRate, fq.FlowMaxRate, fq.Buckets, fq.FlowRefillDelay, fq.LowRateThreshold,
+		"{PacketLimit: %v, FlowPacketLimit: %v, Quantum: %v, InitialQuantum: %v, Pacing: %v, FlowDefaultRate: %v, FlowMaxRate: %v, Buckets: %v, FlowRefillDelay: %v,  LowRateThreshold: %v, Horizon: %v, HorizonDrop: %v}",
+		fq.PacketLimit, fq.FlowPacketLimit, fq.Quantum, fq.InitialQuantum, fq.Pacing, fq.FlowDefaultRate, fq.FlowMaxRate, fq.Buckets, fq.FlowRefillDelay, fq.LowRateThreshold, fq.Horizon, fq.HorizonDrop,
 	)
 }
 
 func NewFq(attrs QdiscAttrs) *Fq {
 	return &Fq{
-		QdiscAttrs: attrs,
-		Pacing:     1,
+		QdiscAttrs:  attrs,
+		Pacing:      1,
+		HorizonDrop: 255,
```

Comment:
This 255 looks like a well-known default value.
Can we define a constant for it?

Reply:
Thanks @aboch. Does `HORIZON_DROP_UNSPEC` work? Or `HorizonDropUnspec`? (I see that both conventions are used for constants.)

Reply:
As you noticed, I see the C style naming of constant all over. I also think this helps cross checking with the Linux code.
`HORIZON_DROP_UNSPEC ` sounds good or whatever #define Linux has for this.

Reply:
There's no name in Linux. In `iproute` the names which are parsed from the command line are `horizon_drop` (1) and `horizon_cap` (0), and the magic number 255 is used as is (it can be whatever number which is not going to become a valid policy ever soon).

I will update the patch to use
```
const (
        HORIZON_DROP_POLICY_CAP     = 0
        HORIZON_DROP_POLICY_DROP    = 1
        HORIZON_DROP_POLICY_DEFAULT = 255
)
```